packages/medusa/src/services/payment-provider.js

```diff
@@ -10,21 +10,41 @@ class PaymentProviderService {
   }
 
   /**
-   * Handles incoming jobs.
-   * @param job {{ eventName: (string), data: (any) }}
-   *    eventName - the name of the event to process
-   *    data - data to send to the subscriber
-   *
+   * Creates a payment session with the given provider.
+   * @param {string} providerId - the id of the provider to create payment with
+   * @param {Cart} cart - a cart object used to calculate the amount, etc. from
+   * @return {Promise} the payment session
+   */
+  createSession(providerId, cart) {
+    const provider = this.retrieveProvider(providerId)
+    return provider.createSession(cart)
```

Comment:
Should this not be `createPayment`, if we look at the current state of `BasePaymentService`?

Reply:
You're right 🤦